### ISSUE TITLE
indexed-db adapter error in source

### ISSUE THREAD

**Issue by MikeTommasi** [brianleroux/lawnchair#196]
the indexed db adapter on the lawnchair home page downloads section does not work (uses deprecated constants, has fixed test name for store). 

The source in github works. 

You should update thev version on the lawnchair downloads....

cheers

MT

**Comment by demaniak** [comment]
Yes please, that would be very nice!

**Comment by mikegioia** [comment]
Yes, *please* update this. I spent about an hour scratching my head as to why Lawnchair wasn't working in Firefox! I don't think any of the source files on the website are up to date.

**Comment by MarkMYoung** [comment]
I see what you are talking about on [developer.mozilla.org](https://developer.mozilla.org/en-US/docs/Web/API/IDBTransaction#Constants).  However, the following line of code checks whether `READ_WRITE` is available and uses the new string constant if it is not.
```javascript
var READ_WRITE = (getIDBTransaction() && 'READ_WRITE' in getIDBTransaction()) ? getIDBTransaction().READ_WRITE : 'readwrite';
```
I don't use IndexedDB so, all of you who do use it are saying this line does not work?

**Comment by MarkMYoung** [comment]
The thing is, I haven't used Firefox in over ten years and I don't use IndexedDB at work so I can test this in short order.  The code Brian has is what Mozilla recommends and it looks good to me; however, I have seen browsers stub in enough stuff to make it look like it has a feature when it does not (e.g. Safari acting like it has toLocaleString for Number and Date).  So, someone with this problem please go to the console in Firefox and tell us what the value of `'READ_WRITE' in getIDBTransaction()` is and what the value of the whole line is.  If it says `true`, that's a Firebox mistake; if it says `false`, then I don't understand how there is a bug since the adapter would be using the string constant.

**Comment by mikegioia** [comment]
I think there are two separate issues in here. Lawnchair and the indexed-db adapter both work for me in Firefox 36 (current latest). There's no problem with the library in my mind.

The issue is that the [code on the webpage](http://brian.io/lawnchair/downloads/lawnchair-adapter-indexed-db-0.6.1.js) is not the same or as current as the [code in Github](https://raw.githubusercontent.com/brianleroux/lawnchair/master/src/adapters/indexed-db.js). The old code on the website caused errors in Firefox that were resolved just by using the Github code. I was just asking if the site owner/operator could update the code on brian.io or just remove it and link directly to Github.

**Comment by demaniak** [comment]
@mikegioia , @MarkMYoung - yes indeed, according to what I have scanned there has been quite a bit of work done on github source, that has not yet made it through to the "official" releases on the lawnchair downloads page. Speaking under correction, this is not just indexed-db related code.
